spec/Staking.md

```diff
@@ -351,6 +351,40 @@ share of "0xAB..CD" = Math.floor(35 * (1000 / (500 + 1000))) = 23
 remaing share (of author) = 110 - (11 + 23) = 76
 ```
 
+## ChangeParameters
+This transaction will change the common parameters when more than half of the stakeholders agree.
+It does not change the other fields of the scheme file, because there are fields related to the genesis block.
+
+It does not provide a feature of voting.
+The vote initiator should collect the signatures through the off-chain.
+
+This transaction increases the `seq` of `Metadata` and changes the `params` of `Metadata`.
+
+### Action
+`[ 0xFF, metadata_seq, new_parameters, ...signatures ]`
+
+#### metadata_seq
+The transaction fails if it's different from the `seq` of `Metadata`.
+It's introduced to prevent replay attacks.
```

Comment:
The transaction fails if the metadata_seq is different from the `seq` of `Metadata` and is introduced to prevent replay attacks.